Userland/Libraries/LibGfx/ICCProfile.h

```diff
@@ -335,6 +335,25 @@ class TextTagData : public TagData {
     String m_text;
 };
 
+// ICC v4, 10.31 XYZType
+class XYZTagData : public TagData {
+public:
+    static constexpr TagTypeSignature Type { 0x58595A20 }; // 'XYZ '
```

Comment:
Trailing space in `'XYZ '` probably not intended?

Reply:
It is intended. It's a "four character code ('fourcc')", meaning a 32-bit value that looks readable in a hex dump. XYZ are just 3 values, so they used a space as fourth. See how the hex number ends in `20`, and 0x20 is a space. (And 0x58 is an X, etc)